admin/templates/osf_groups/osf_groups.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block title %}
+{% load node_extras %}
+<title>OSF Group</title>
+{% endblock title %}
+{% block content %}
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="btn-group" role="group">
+                <a href="{% url 'osf_groups:search' %}" class="btn btn-default"><i class="fa fa-search"></i></a>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <h3>OSF Group details</h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <table class="table table-striped">
+                <thead>
+                    <tr>
+                        <th>Field</th>
+                        <th>Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>_id</td>
+                        <td colspan="2">{{ group.named_id }}</td>
+                    </tr>
+                    <tr>
+                        <td>Name</td>
+                        <td colspan="2">{{ group.name }}</td>
+                    </tr>
+                    <tr>
+                        <td>Date Created</td>
+                        <td colspan="2">{{ group.created }}</td>
+                    </tr>
+                    <tr>
+                        <td>Date Modified</td>
+                        <td colspan="2">{{ group.modified }}</td>
+                    </tr>
+                    <tr>
+                        <td>Creator</td>
+                        <td colspan="2"><a href="{{ group.creator.id | reverse_user }}">{{ group.creator }}</a></td>
```

Comment:
Once you've made the changes to the groups serializer, you will need to change several references in this template.
```html
  <td colspan="2"><a href="{{ group.creator.id | reverse_user }}">{{ group.creator.name }}</a></td>
``` 
, for example